nerdlandbot/scheduler/ChurchScheduler.py

```diff
@@ -0,0 +1,37 @@
+import os
+import discord
+import requests
+import time
+
+from datetime import datetime, timedelta
+from discord.ext import tasks
+
+from nerdlandbot.commands.GuildData import get_all_guilds_data, GuildData, get_guild_data
+from nerdlandbot.translations.Translations import get_text as translate
+from nerdlandbot.helpers.TranslationHelper import get_culture_from_context as culture
+from nerdlandbot.helpers.emoji import fist, church_emoji
+from nerdlandbot.helpers.log import warn
+
+@tasks.loop(minutes=1.0)
+async def church_fights(bot):
+	current_time = datetime.now()
+	if current_time.hour == 16 and current_time.minute == 0:
+		guilds_data = await get_all_guilds_data()
+		day = current_time.day
+		for guild_data in guilds_data:
+			channel_id = guild_data.church_channel
+			try:
+				channel = bot.get_channel(channel_id)
+			except:
+				warn("Something went wrong. It's possible the channel is not set.")
```

Comment:
You're catching an exception but later using the variable set inside the try part (line 36)

And while you're at it, please could you change the tabs to spaces to follow the repo's convention? :)